backend/views.py

```diff
@@ -225,6 +225,14 @@ def get(self, request: Request,
 
                 return Response(result, status=status.HTTP_200_OK)
 
+            elif "result-name" == string:
+                result["all"] = list(RecruitmentResult.objects.
+                                     order_by().
+                                     values_list('result', flat=True).
+                                     distinct())
+
+                return Response(result, status=status.HTTP_200_OK)
+
```

Comment:
To chyba nie do tego brancha, tak? Tak bardziej na przyszłość, żeby nie dawać różnych rzeczy do wspólnego brancha (nawet jeśli to ma tu być, to tak tylko mówię ;) )